### PR TITLE
CU-860pqvpud - feat: install ci cluster as part of agent release

### PR DESCRIPTION
**Pull request by nirbenator**
* adds ci cluster (already done manually)
* brings back validation for agent release
* adds connection of staging agent to prod (already done manually)

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -98,3 +98,18 @@ steps:
             API_KEY: /komodor/production/kubernetes/api-key
     label: "Install new watcher version on komodor production :kubernetes: cluster"
     if: build.message !~ /feat\(helm-dashboard\):/i && build.branch == "master"
+
+  - commands:
+      - aws eks --region us-east-1 update-kubeconfig --name ci
```
Comment (MickaelAlliel) [inline .buildkite/pipeline.yml:103]:
```suggestion
      - komo ctx ci
```

**Review by MickaelAlliel — APPROVED** [review]

**Review by MickaelAlliel — APPROVED** [review]

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -98,3 +101,18 @@ steps:
             API_KEY: /komodor/production/kubernetes/api-key
     label: "Install new watcher version on komodor production :kubernetes: cluster"
     if: build.message !~ /feat\(helm-dashboard\):/i && build.branch == "master"
+
+  - commands:
+      - komo ctx ci
+      - helm repo add komodorio https://helm-charts.komodor.io
+      - helm repo update
+      - helm upgrade --install k8s-watcher komodorio/k8s-watcher --reuse-values --dry-run
+      - helm upgrade --install k8s-watcher komodorio/k8s-watcher --set apiKey=$$API_KEY --set imagePullSecret=docker-cfg-komodorio --set watcher.telemetry.enable=true --set watcher.collectHistory=true --set watcher.nameDenylist="{leader,election}" --set watcher.resources.secret=true --set watcher.redact="{.*KEY.*,.*key.*,.*BUGSNAG.*}" --set watcher.enableAgentTaskExecution=true --set watcher.enableAgentTaskExecutionV2=true --set watcher.allowReadingPodLogs=true --set watcher.actions.basic=true --set watcher.actions.advanced=true --set watcher.enableHelm=true --set helm.enableActions=true --set watcher.actions.podExec=true --set metrics.enabled=true
```
[the remark this answers is not in the document]
Comment (nirbenator) [reply .buildkite/pipeline.yml:110]: i already did :)

[the remark this answers is not in the document]
Comment (nirbenator) [reply .buildkite/pipeline.yml:110]: image

**Review by Moran-k — APPROVED** [review]

**Comment by nirbenator** [comment]
CU-860qbhuad

**Comment by nirbenator** [comment]
public: internal pipeline changes

**Pull request merged**